frontend/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,29 @@
+import './HomePage.css';
+import RecipesPage from "./RecipesPage.tsx";
+import {Recipe} from "../types/Recipe.ts";
+
+type HomePageProps = {
+    recipes: Recipe[];
+}
+
+export default function HomePage(props: Readonly<HomePageProps>){
+    return (
+        <div className="homepage-container">
+            <img
+                src="/images/homepagePhoto.jpeg"
+                alt="Homepage"
+                className="homepage-image"
+            />
+            <RecipesPage recipes={props.recipes}/>
+
+            <br/>
+            <br/>
+            <br/>
+            <br/>
+            <br/>
```

Comment:
I think working with margins/paddings in the CSS yould be better than adding empty lines to create space between components. This could cause problems when you resize the browser window

Reply:
Thx for your review and for the great suggestions. :) Unfortunately I didn't manage to achieve what I want only with the margin for the moment, because the footer has a hide and appear feature, and without these empty lines it could prevent some of the content to be seen. But improvement is on todo list ;)